libmscore/layout.cpp

```diff
@@ -3727,13 +3727,11 @@ System* Score::collectSystem(LayoutContext& lc)
                   break;
                   }
             }
-      if (!allShown) {
```

Comment:
Will this change reintroduce issues #278057 and #277666? 
If not, also the few lines above should be removed (3724-3729).

Reply:
Checked #278057 and #277666
No issues found